### ISSUE TITLE
Short hg revisions don't show up in Treeherder

### ISSUE THREAD

**Issue by JohanLorenzo** [mozilla-releng/fennec-aurora-task-creator#9]
Sometimes (probably when builds are re-spun), tasks are created with short hg revisions (see [this one for instance](https://tools.taskcluster.net/task-inspector/#c60sevnMQ6S11q0xuTB4dw/)). The tasks are correctly uploaded to Treeherder, but they don't appear on the dashboard. 

One way to fix that: get the long revision from hg.m.o.

**Comment by JohanLorenzo** [comment]
For reference, logs gotten with short revision:
```
2016-11-29T11:07:37.861673+00:00 app[worker.1]: 2016-11-29 11:07:37,861 - worker.py - DEBUG - Got a new message from the queue. Body: {'workerId': 'buildbot', 'status': {'taskGroupId': 'BQzruPklRSOvdKQTuZ9SZA', 'runs': [{'scheduled': '2016-11-29T11:06:43.785Z', 'workerId': 'buildbot', 'takenUntil': '2016-11-29T11:27:35.594Z', 'runId': 0, 'state': 'completed', 'started': '2016-11-29T11:06:44.899Z', 'workerGroup': 'buildbot', 'reasonCreated': 'scheduled', 'resolved': '2016-11-29T11:07:37.604Z', 'reasonResolved': 'completed'}], 'state': 'completed', 'workerType': 'buildbot', 'taskId': 'BQzruPklRSOvdKQTuZ9SZA', 'deadline': '2016-11-29T12:06:42.618Z', 'retriesLeft': 5, 'provisionerId': 'null-provisioner', 'schedulerId': '-', 'expires': '2017-11-29T12:06:42.618Z'}, 'runId': 0, 'workerGroup': 'buildbot', 'version': 1}
2016-11-29T11:07:38.265532+00:00 app[worker.1]: 2016-11-29 11:07:38,265 - worker.py - INFO - Processing revision "94dac74c74a6" from repository "mozilla-aurora" (triggered by completed task "BQzruPklRSOvdKQTuZ9SZA")
2016-11-29T11:07:39.238351+00:00 app[worker.1]: 2016-11-29 11:07:39,238 - syncclient.py - WARNING - Retrying because of Indexed task not found - {
2016-11-29T11:07:39.238365+00:00 app[worker.1]:     "error": {
2016-11-29T11:07:39.238366+00:00 app[worker.1]:         "namespace": "gecko.v2.mozilla-aurora.revision.94dac74c74a6.mobile-l10n.android-x86-opt.multi"
2016-11-29T11:07:39.238367+00:00 app[worker.1]:     },
2016-11-29T11:07:39.238368+00:00 app[worker.1]:     "message": "Indexed task not found"
2016-11-29T11:07:39.238372+00:00 app[worker.1]: }
2016-11-29T11:07:39.749057+00:00 app[worker.1]: 2016-11-29 11:07:39,748 - syncclient.py - WARNING - Retrying because of Indexed task not found - {
2016-11-29T11:07:39.749069+00:00 app[worker.1]:     "error": {
2016-11-29T11:07:39.749070+00:00 app[worker.1]:         "namespace": "gecko.v2.mozilla-aurora.revision.94dac74c74a6.mobile-l10n.android-x86-opt.multi"
2016-11-29T11:07:39.749070+00:00 app[worker.1]:     },
2016-11-29T11:07:39.749071+00:00 app[worker.1]:     "message": "Indexed task not found"
2016-11-29T11:07:39.749075+00:00 app[worker.1]: }
2016-11-29T11:07:40.472575+00:00 app[worker.1]: 2016-11-29 11:07:40,472 - syncclient.py - WARNING - Retrying because of Indexed task not found - {
2016-11-29T11:07:40.472579+00:00 app[worker.1]:     "error": {
2016-11-29T11:07:40.472580+00:00 app[worker.1]:         "namespace": "gecko.v2.mozilla-aurora.revision.94dac74c74a6.mobile-l10n.android-x86-opt.multi"
2016-11-29T11:07:40.472581+00:00 app[worker.1]:     },
2016-11-29T11:07:40.472582+00:00 app[worker.1]:     "message": "Indexed task not found"
2016-11-29T11:07:40.472586+00:00 app[worker.1]: }
2016-11-29T11:07:41.132243+00:00 app[worker.1]: 2016-11-29 11:07:41,131 - syncclient.py - WARNING - Retrying because of Indexed task not found - {
2016-11-29T11:07:41.132255+00:00 app[worker.1]:     "error": {
2016-11-29T11:07:41.132256+00:00 app[worker.1]:         "namespace": "gecko.v2.mozilla-aurora.revision.94dac74c74a6.mobile-l10n.android-x86-opt.multi"
2016-11-29T11:07:41.132257+00:00 app[worker.1]:     },
2016-11-29T11:07:41.132258+00:00 app[worker.1]:     "message": "Indexed task not found"
2016-11-29T11:07:41.132264+00:00 app[worker.1]: }
2016-11-29T11:07:42.255949+00:00 app[worker.1]: 2016-11-29 11:07:42,255 - syncclient.py - WARNING - Retrying because of Indexed task not found - {
2016-11-29T11:07:42.255962+00:00 app[worker.1]:     "error": {
2016-11-29T11:07:42.255963+00:00 app[worker.1]:         "namespace": "gecko.v2.mozilla-aurora.revision.94dac74c74a6.mobile-l10n.android-x86-opt.multi"
2016-11-29T11:07:42.255964+00:00 app[worker.1]:     },
2016-11-29T11:07:42.255965+00:00 app[worker.1]:     "message": "Indexed task not found"
2016-11-29T11:07:42.256021+00:00 app[worker.1]: }
2016-11-29T11:07:44.260854+00:00 app[worker.1]: 2016-11-29 11:07:44,260 - worker.py - INFO - Revision "94dac74c74a6" does not have architecture "android-x86-opt" completed yet
2016-11-29T11:07:44.260996+00:00 app[worker.1]: 2016-11-29 11:07:44,260 - worker.py - INFO - Marking message as read. Processed task "BQzruPklRSOvdKQTuZ9SZA"
2016-11-29T11:09:24.441874+00:00 app[worker.1]: 2016-11-29 11:09:24,441 - worker.py - DEBUG - Got a new message from the queue. Body: {'workerId': 'buildbot', 'status': {'taskGroupId': 'dVj29cbZQuSSeXh6MSm91w', 'runs': [{'scheduled': '2016-11-29T11:07:39.886Z', 'workerId': 'buildbot', 'takenUntil': '2016-11-29T11:29:21.477Z', 'runId': 0, 'state': 'completed', 'started': '2016-11-29T11:07:41.131Z', 'workerGroup': 'buildbot', 'reasonCreated': 'scheduled', 'resolved': '2016-11-29T11:09:24.195Z', 'reasonResolved': 'completed'}], 'state': 'completed', 'workerType': 'buildbot', 'taskId': 'dVj29cbZQuSSeXh6MSm91w', 'deadline': '2016-11-29T12:07:38.730Z', 'retriesLeft': 5, 'provisionerId': 'null-provisioner', 'schedulerId': '-', 'expires': '2017-11-29T12:07:38.730Z'}, 'runId': 0, 'workerGroup': 'buildbot', 'version': 1}
2016-11-29T11:09:24.693090+00:00 app[worker.1]: 2016-11-29 11:09:24,692 - worker.py - INFO - Processing revision "94dac74c74a6" from repository "mozilla-aurora" (triggered by completed task "dVj29cbZQuSSeXh6MSm91w")
2016-11-29T11:09:25.048680+00:00 app[worker.1]: 2016-11-29 11:09:25,048 - syncclient.py - WARNING - Retrying because of Indexed task not found - {
2016-11-29T11:09:25.048699+00:00 app[worker.1]:     "error": {
2016-11-29T11:09:25.048700+00:00 app[worker.1]:         "namespace": "gecko.v2.mozilla-aurora.revision.94dac74c74a6.mobile-l10n.android-x86-opt.multi"
2016-11-29T11:09:25.048701+00:00 app[worker.1]:     },
2016-11-29T11:09:25.048702+00:00 app[worker.1]:     "message": "Indexed task not found"
2016-11-29T11:09:25.048707+00:00 app[worker.1]: }
2016-11-29T11:09:26.806296+00:00 app[worker.1]: 2016-11-29 11:09:26,806 - publish.py - DEBUG - Valid artifacts candidates found: ['public/build/fennec-52.0a2.multi.android-i386.apk']
2016-11-29T11:09:26.806395+00:00 app[worker.1]: 2016-11-29 11:09:26,806 - publish.py - DEBUG - Valid artifacts candidates found: ['public/build/fennec-52.0a2.multi.android-arm.apk']
2016-11-29T11:09:28.876872+00:00 app[worker.1]: 2016-11-29 11:09:28,876 - publish.py - INFO - Created task c60sevnMQ6S11q0xuTB4dw: {'status': {'taskGroupId': 'c60sevnMQ6S11q0xuTB4dw', 'runs': [{'reasonCreated': 'scheduled', 'scheduled': '2016-11-29T11:09:28.609Z', 'runId': 0, 'state': 'pending'}], 'state': 'pending', 'workerType': 'pushapk-v1-dev', 'taskId': 'c60sevnMQ6S11q0xuTB4dw', 'deadline': '2016-11-29T12:09:27.229Z', 'retriesLeft': 0, 'provisionerId': 'scriptworker-prov-v1', 'schedulerId': '-', 'expires': '2017-11-29T12:09:27.229Z'}}
2016-11-29T11:09:28.876998+00:00 app[worker.1]: 2016-11-29 11:09:28,876 - worker.py - INFO - Marking message as read. Processed task "dVj29cbZQuSSeXh6MSm91w"
```

**Comment by JohanLorenzo** [comment]
Closed because project is now retired (see [bug 1353303](https://bugzilla.mozilla.org/show_bug.cgi?id=1353303))
